### ISSUE TITLE
Update Databases / Add New Databases

### ISSUE THREAD

**Issue by smahaffey** [KechrisLab/multiMiR#45]
We will start work on updating:
miRTarBase – v7 -> v9
TargetScan – v7.2 -> v8

Are there any databases you would like to request?  Please comment.

**Comment by tracywan9** [comment]
Thanks a lot for your upcoming update！

**Comment by EvenZhang-spec** [comment]
Thinks,mirWalk?

**Comment by chemy-cm** [comment]
Thanks a lot. By the way, Has the database been updated so far?

**Comment by knokknok** [comment]
Would it be possible to have a status update?
🙏

**Comment by jeremydavisturak** [comment]
Is it possible to download a snapshot of all the data in the database (both before and after the update)?

**Comment by smahaffey** [comment]
@jeremydavisturak The updated database will be a new complete version so the previous versions will all still be available for the purpose of allowing replication of searches and previous results.  Any database version can still be used.  The R package provides functionality to select the database version for queries.

As far as downloading the full database, we will have to discuss it.  I know at least one of the miRNA-Target databases had a restriction on allowing downloading data from us directly even thought they granted permission to incorporate it into MultiMiR.  I'll add it to the list for discussion with the group.

**Comment by mattbcvs** [comment]
Thanks for the great tool, providing a lot of avenues for our lab.

Some comments around the entries for "validated" databases:
- Source_type column entries are unclear to me, why are some interactions "functional" and others "functional(weak)"?
- Experiment category naming looks a bit inconsistent e.g. "immunoblotting" and "immunoblot analysis", further some are seperated by a "//" and others by ";" which can add confusion.

Would help to make more user-friendly, thanks again for the tool though!

**Comment by smahaffey** [comment]
@mattbcvs Thank you for the feedback.  I'm getting started with the update now.  So I'll try to keep these in mind.  For these issues they are from the original source.  I believe they did have a reason for the functional vs functional(weak) interactions.  So we don't want to alter the original meaning assigned to those distinctions as we're summarizing them across sources.  

I do feel as though I could look at the separator issue and try to make everything consistent with this update.  I will review each database and see what I can do so everything is slightly more consistent without altering the original data provided by these sources.

**Comment by smahaffey** [comment]
The database update is nearly finished.  I have updated the 3 existing databases Tarbase v9, miRTarbase v9,  and TargetScan v8.  

Tarbase will now include a list of the PubMed ids for each positive/negative entry.  Those were never imported into the database correctly.

I'm just revisiting the ID issues that exist with old annotations that have changed since some of the older databases were released.  After I can clean them up further I will release this database update, hopefully before the end of July.

The issues with the R package will be next and then we will revisit including new databases and improvements.

Thank you for your patience.
